### PR TITLE
[EDU-5142] fix: adjust WAF custom rules note

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue: 

[EDU-5142]

### Changes

- Adjust WAF custom rules note.

### Additional links

n/a.

[EDU-5142]: https://aziontech.atlassian.net/browse/EDU-5142?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by GabrielAzion — APPROVED** [review]

**Review by gabriel-azion — APPROVED** [review]

File: src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/custom-allowed-rule.mdx [hunk 1]
```diff
@@ -93,7 +93,7 @@ See the list of all available internal rules below:
 | 1500    | Possible File Upload attempt: `asp/php` or `.ph`, `.asp`, `.ht` found in filename in a multipart POST containing a file.                   |
 
 :::caution[Warning]
-Requests that fall under rules 1 to 18 might be blocked in some cases, even if the WAF is operating in *learning* mode. Read the [Rules Engine for Edge Firewall](/en/documentation/products/secure/edge-firewall/rules-engine/) documentation for the definition of learning/blocking modes. Read this guide [how to check your WAF mode](/en/documentation/products/guides/how-to-check-your-waf-mode/).
```
Comment (arturdj) [inline src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/custom-allowed-rule.mdx:96]: Na frase, a única regra em que isso se aplica, agora, é a 13.

> "Requests that fall under rule **13** will be blocked in some cases (...)"

**Review by arturdj — CHANGES_REQUESTED** [review]
Tem um erro ainda na frase.

File: src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/custom-allowed-rule.mdx [hunk 2]
```diff
@@ -93,7 +93,7 @@ See the list of all available internal rules below:
...
+Requests that fall under rules 1 to 18 will be blocked in some cases, even if the WAF is operating in *learning* mode. Read the [Rules Engine for Edge Firewall](/en/documentation/products/secure/edge-firewall/rules-engine/) documentation for the definition of learning/blocking modes. Read this guide [how to check your WAF mode](/en/documentation/products/guides/how-to-check-your-waf-mode/).
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/custom-allowed-rule.mdx:96]:
```suggestion
Requests that fall under rule 13 will be blocked in some cases, even if the WAF is operating in *learning* mode. Read the [Rules Engine for Edge Firewall](/en/documentation/products/secure/edge-firewall/rules-engine/) documentation for the definition of learning/blocking modes. Read this guide [how to check your WAF mode](/en/documentation/products/guides/how-to-check-your-waf-mode/).
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/web-application-firewall/waf-custom-allowed-rules.mdx [hunk 3]
```diff
@@ -91,7 +91,7 @@ Veja, abaixo, a lista de todas as regras internas disponíveis:
 | 1500 | Possible File Upload attempt: `asp/php` ou `.ph`, `.asp`, `.ht` encontrados no nome de um arquivo em um `POST` de múltiplas partes contendo um arquivo |
 
 :::caution[atenção]
-Requisições que se enquadram nas regras de 1 a 18 podem ser bloqueadas em alguns casos, mesmo que o WAF esteja operando em modo *learning*. Veja na documentação do [Rules Engine para Edge Firewall](/pt-br/documentacao/produtos/secure/edge-firewall/rules-engine/) a definição dos modos learning/blocking. Veja no guia [como verificar o modo do seu WAF](/pt-br/documentacao/produtos/guias/como-verificar-modo-do-seu-waf/).
+Requisições que se enquadram nas regras de 1 a 18 serão bloqueadas em alguns casos, mesmo que o WAF esteja operando em modo *learning*. Veja na documentação do [Rules Engine para Edge Firewall](/pt-br/documentacao/produtos/secure/edge-firewall/rules-engine/) a definição dos modos learning/blocking. Veja no guia [como verificar o modo do seu WAF](/pt-br/documentacao/produtos/guias/como-verificar-modo-do-seu-waf/).
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/web-application-firewall/waf-custom-allowed-rules.mdx:94]:
```suggestion
Requisições que se enquadram na regra 13 serão bloqueadas em alguns casos, mesmo que o WAF esteja operando em modo *learning*. Veja na documentação do [Rules Engine para Edge Firewall](/pt-br/documentacao/produtos/secure/edge-firewall/rules-engine/) a definição dos modos learning/blocking. Veja no guia [como verificar o modo do seu WAF](/pt-br/documentacao/produtos/guias/como-verificar-modo-do-seu-waf/).
```

**Review by arturdj — APPROVED** [review]

**Pull request merged**